sdks/python/apache_beam/examples/complete/game/game_stats.py

```diff
@@ -71,7 +71,6 @@
 """
 
 from __future__ import absolute_import
-from __future__ import division
```

Comment:
I do not think you need to remove this. Please fix it in a future PR.

Reply:
Done